README.md

```diff
@@ -48,6 +48,8 @@ To get started (with Intellij Ultimate):
 4. In the frontend folder, use ``npm install``.
 
 Happy Hacking!
+### Tests
```

Comment:
ah sehr gut. Bei dir kann ich mal spicken, was du so in deine Readme schreibst :)